File: src/main/java/coza/trojanc/stonewriter/template/process/ProcessedTemplate.java
package coza.trojanc.stonewriter.template.process;

import coza.trojanc.stonewriter.template.process.fields.ProcessedLineItem;

import java.util.ArrayList;
import java.util.List;

/**
 * A class representing a template that has been processed
 * Created by Charl on 2016-10-11.
 */
public class ProcessedTemplate {

	private List<ProcessedLineItem> items = new ArrayList<>();

	public List<ProcessedLineItem> getItems() {
		return items;
	}

	public void setItems(List<ProcessedLineItem> items) {
		this.items = items;
	}
}

File: src/main/java/coza/trojanc/receipt/template/process/ProcessedTemplate.kt
package coza.trojanc.receipt.template.process

import coza.trojanc.receipt.template.process.fields.ProcessedLineItem

import java.util.ArrayList

/**
 * A class representing a template that has been processed
 * @author Charl Thiem
 */
class ProcessedTemplate {

    private var items: MutableList<ProcessedLineItem> = ArrayList()

    fun getItems(): List<ProcessedLineItem> {
        return items
    }

    fun setItems(items: List<ProcessedLineItem>) {
        this.items.clear();
        this.items.addAll(items);
    }

    fun addProcessedLineItem(processedLineItem: ProcessedLineItem){
        this.items.add(processedLineItem);
    }
}
